administration/src/mui-modules/application/primitive-inputs/DateForm.tsx

```diff
@@ -40,28 +43,29 @@ const DateForm: Form<State, ValidatedInput, AdditionalProps, Options> = {
     const [touched, setTouched] = useState(false)
     const { showAllErrors, disableAllInputs } = useContext(FormContext)
     const validationResult = DateForm.validate(state, options)
-
     const isInvalid = validationResult.type === 'error'
 
     return (
-      <TextField
-        variant='standard'
-        fullWidth
-        style={{ margin: '4px 0', minWidth }}
-        type='date'
-        label={label}
-        required
-        inputProps={{
-          max: options.maximumDate ? formatISO(options.maximumDate, { representation: 'date' }) : undefined,
-          min: '1900-01-01',
-        }}
+      <CustomDatePicker
+        date={null}
```

Comment:
🔧 it looks like the `DesktopDatePicker` from mui also accepts `undefined`
So you could remove this prop here and adjust the `customDatePickerProps`
`date?: Date`